app/models/katello/host_status_manager.rb

```diff
@@ -0,0 +1,13 @@
+module Katello
```

Comment:
do you think app/services might be a better place for this?

Reply:
sure